mtags/src/main/scala-2/scala/meta/internal/pc/Signatures.scala

```diff
@@ -138,52 +132,90 @@ trait Signatures { this: MetalsGlobal =>
       nameResolvesToSymbol(top.name.toTermName, top)
     }
 
-    def nameResolvesToSymbol(sym: Symbol): Boolean = {
-      nameResolvesToSymbol(sym.name, sym)
+    def isSymbolInScope(sym: Symbol, prefix: Type = NoPrefix): Boolean = {
+      nameResolvesToSymbol(sym.name, sym, prefix)
     }
-    def nameResolvesToSymbol(name: Name, sym: Symbol): Boolean = {
+    def nameResolvesToSymbol(
+        name: Name,
+        sym: Symbol,
+        prefix: Type = NoPrefix
+    ): Boolean = {
       lookupSymbol(name) match {
-        case Nil => true
-        case lookup => lookup.exists(_.symbol.isKindaTheSameAs(sym))
+        case Nil => false
```

Comment:
This is a change I did here, since if lookup did not find anything then it should be false. Indeed a lot of tests seem to have unneccesarily long names.